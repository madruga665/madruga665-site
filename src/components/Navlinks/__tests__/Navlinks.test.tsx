import { render, screen } from "@testing-library/react";
import NavLinks from "..";

describe("Testes do componente NavLinks", () => {
  test("Snapshot do componente", () => {
    const { container } = render(<NavLinks />);

    expect(container).toMatchSnapshot();
  });
  test("A navBar estÃ¡ sendo renderizada", () => {
    render(<NavLinks />);
    const navBar = screen.getByRole("navigation");
    expect(navBar).toBeInTheDocument();
  });
  test("Quantidade de items de lista", () => {
    render(<NavLinks />);
    const listItems = screen.getAllByRole("listitem");
    expect(listItems.length).toBe(4);
  });
});
