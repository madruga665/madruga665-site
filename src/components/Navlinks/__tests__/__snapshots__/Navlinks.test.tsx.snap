// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testes do componente NavLinks Snapshot do componente 1`] = `
<div>
  <nav
    class="sc-bdvvtL ikFlXf"
  >
    <ul>
      <li>
        <a
          href="#home"
        >
          HOME
        </a>
      </li>
      <li>
        <a
          href="#sobre"
        >
          SOBRE
        </a>
      </li>
      <li>
        <a
          href="#projetos"
        >
          PROJETOS
        </a>
      </li>
      <li>
        <a
          href="#contato"
        >
          CONTATO
        </a>
      </li>
    </ul>
  </nav>
</div>
`;
